### PR TITLE
Update Sidebar Links and Design finished

### PR DESCRIPTION
**Pull request by Bdhawa123**
Update Sidebar Links and Design from Trello 
Finished !!!

## Screenshots/Links

--> Replace this with an screenshot, GIFs if applicable
image

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
     "react-i18next": "^11.3.4",
+    "react-onclickoutside": "^6.9.0",
```
Comment (aviskarkc10) [inline client/package.json:18]: let's remove this dependency now that we are not using it.

File: client/src/components/NavLink.js [hunk 2]
```diff
@@ -50,25 +64,25 @@ class NavMenu extends Component {
                       rel="noopener noreferrer"
                       href="https://about.askco19.com/#lp-pom-block-118"
                     >
-                      {t('navLink:about')}
+                      {t('navLink:ABOUT')}
```
Comment (aviskarkc10) [inline client/src/components/NavLink.js:67]: I see you have changed the value here, but not on the json files where we store translation values.

Were these changes necessary? Why are we making this change? Or were there conflicts and the change got reverted?

File: client/src/components/NavLink.js [hunk 3]
```diff
@@ -86,9 +100,20 @@ class NavMenu extends Component {
...
+                      {t('navLink:CONTACT US')}
                     </a>
                   </li>
+                  <li>
+                    <span rel="noopener noreferrer">
+                      {t('navLink:LANGUAGE')}
+                    </span>
+                    <div>
+                      <LanguageSelector
+                        Dropdownbox={{ backgroundColor: '#1678c2' }}
```
Comment (aviskarkc10) [inline client/src/components/NavLink.js:112]: let's use class names instead of inline styling. We could create a class `sidebar-dropdown` or `sidebar-language-dropdown` where we put this particular style

File: client/src/styles/NavLink.css [hunk 4]
```diff
@@ -46,6 +47,10 @@ nav ul li a {
   letter-spacing: 0.3px;
 }
 
+#LanguageSelector {
```
Comment (aviskarkc10) [inline client/src/styles/NavLink.css:50]: use class instead of id for this. it's a better practice

Comment (Bdhawa123) [reply client/src/styles/NavLink.css:50]: I forgot to get rid of that we're not using it, I'll create an object and pass on as a props as you suggested

File: client/src/components/NavLink.js (hunk 2 above)
Comment (Bdhawa123) [reply client/src/components/NavLink.js:67]: Few of the changes were requested on the trello card, I'll update the files accordingly.

File: client/src/components/SearchBar.js [hunk 5]
```diff
@@ -2,6 +2,8 @@ import _ from 'lodash';
 import React from 'react';
 import { Search } from 'semantic-ui-react';
 import { withTranslation } from 'react-i18next';
+// eslint-disable-next-line no-unused-vars
+import styles from '../styles/SearchBar.css';
```
Comment (aviskarkc10) [inline client/src/components/SearchBar.js:6]:
```suggestion
import '../styles/SearchBar.css';
```

You can just do this and won't need to use the variable or the eslint rule

File: client/src/components/SearchBar.js [hunk 6]
```diff
@@ -2,6 +2,8 @@ import _ from 'lodash';
 import React from 'react';
 import { Search } from 'semantic-ui-react';
 import { withTranslation } from 'react-i18next';
+// eslint-disable-next-line no-unused-vars
```
Comment (aviskarkc10) [inline client/src/components/SearchBar.js:5]: let's also get rid of this. and i will merge the PR.
